src/api.js

```diff
@@ -505,7 +515,7 @@ builder.declare({
     } catch (e) {
       // 403 from Github indicates this integration doesn't have permission to post this status,
       // so return that on to the user
-      if (e.code == 403) {
+      if (e.code === 403) {
```

Comment:
it'd be good to land all of these fixes (===, extra semicolons, etc.) in a single PR.  I suspect your editor is auto-fixing these?  They're distracting and have the potential to cause merge conflicts when mixed into other, more substantive PRs.

Reply:
No, it's me who auto-fixes these. Yeah, I can totally land these in a separate PR (I'll throw couple of other similar things in that PR then, actually)

Reply:
Adding couple of rules to `eslint-config-taskcluster` seems like an easy thing to do? I can do that